etc/mock_data_dir/cat1/fuction_select/pretty_count_star.json

```diff
@@ -0,0 +1,21 @@
+{
+  "results": [
+    {
+      "series": [
+        {
+          "name": "average_temperature",
+          "columns": [
+            "time",
+            "count_degrees"
+          ],
+          "values": [
+            [
+              0,
+              15258
+            ]
+          ]
+        }
+      ]
+    }
+  ]
+}
```

Comment:
What's the purpose of `pretty_count_star`? If it's just to indicate that the .json files don't care about whitespace, then I think it ought to be different from an existing query/result.